### DISCUSSION TITLE
KeyMapping

### DISCUSSION THREAD

**Discussion by Neo3477** [nikitabobko/AeroSpace#729] [questions-and-answers] [answered]
Hello everyone,

my keymapping is set to qwerty, but I use a qwertz keyboard. Now I am trying to find the exact name of the key for a certain function, so far without success....  
 
alt-shift-slash = 'resize smart -50'
alt-shift-rightbracket = 'resize smart +50'

Slash is minus for me, which also works. But not with the square bracket on the right, which would be plus for me, I can't get any further.

I have already tested the following spellings:

rightbracket
bracketright
right-bracket
bracket-right
right_bracket
bracket_right

I am grateful for any help....

**Comment by Dimentium** [comment, marked as answer]
Based on the file Sources/AppBundle/config/keysMap.swift, you should try 'rightSquareBracket'

**Reply by Neo3477** [reply]
That worked, thank you very much.
